src/features/situation-builder/api/recommend-situations.ts

```
@@ -0,0 +1,24 @@
+import { useAuthApiClient } from "@/lib/auth/useAuthApiClient";
+import { TRecommend } from "@/types/api";
+import { queryOptions, useMutation } from "@tanstack/react-query";
```

ESLint

[error] 'queryOptions' is defined but never used.

(@typescript-eslint/no-unused-vars)